keepalived/check/check_daemon.c

```diff
@@ -253,7 +253,11 @@ reload_check_thread(__attribute__((unused)) thread_t * thread)
 	thread_cleanup_master(master);
 	free_global_data(global_data);
 
-	free_checkers_queue();
+	/* Save previous checker data */
+	old_checkers_queue = checkers_queue;
+	checkers_queue = NULL;
+	ncheckers = 0;
```

Comment:
Is it necessary to set this to 0? Does it create a risk of two checkers having the same id?